### ISSUE TITLE
Support parallels driver on apple m1

### ISSUE THREAD

**Issue by Patrick0308** [kubernetes/minikube#12116]
**Steps to reproduce the issue:**
On Apple m1, do this command
```bash
❯ minikube start -p test --driver=parallels

😄  [test] minikube v1.22.0 on Darwin 11.4 (arm64)
✨  Using the parallels driver based on user configuration

❌  Exiting due to DRV_UNSUPPORTED_OS: The driver 'parallels' is not supported on darwin/arm64
```
Parallels support for darwin/arm64, see https://www.parallels.com/blogs/parallels-desktop-apple-silicon-mac/

**Full output of `minikube logs` command:**
<details>

</details>

**Full output of failed command:**
<details>

</details>

**Comment by afbjorklund** [comment]
This is being discussed in https://github.com/Parallels/docker-machine-parallels/issues/99

It is currently blocked by #9228 (there is no minikube.iso for arm64 yet)

**Comment by afbjorklund** [comment]
Duplicate of #11219

**Comment by spowelljr** [comment]
As @afbjorklund pointed out, this is a duplicate of #11219. To keep discussion centralized I'm going to close this issue, however, please follow along with #11219 and feel free to contribute to any discussion there, thank you!
